chore(docs): Update liveness troubleshooting guide with connection timeout error (#5320)
aws-amplify/amplify-ui: docs/src/pages/[platform]/connected-components/liveness/troubleshooting/index.page.mdx

Make sure you are using a supported AWS region. Please refer to [Liveness supported regions](https://aws.amazon.com/rekognition/face-liveness/#FAQs) documentation.

### Error: Connection time out

> The websocket connection to the Rekognition streaming service will time out after 5 seconds if a connection is not made. This often happens due to slow network speeds. If you encounter this issue, try using a different network or testing your network latency.

Reviewer: ```suggestion
The websocket connection to the Rekognition streaming service will time out after 5 seconds if a connection is not made. This often happens due to a slow network connection. If you encounter this issue, try using a different network or testing your network latency.
```

> The websocket connection to the Rekognition streaming service will time out after 5 seconds if a connection is not made. This often happens due to a slow network connection. If you encounter this issue, try using a different network or testing your network latency.

Reviewer 2: Since this documentation is targeting developers rather than users, should we change this messaging? Maybe something like this?

```suggestion
The websocket connection to the Rekognition streaming service will time out after 5 seconds if a connection is not made. This often happens due to a slow network connection. If your users encounter this issue, prompt them to try again at a different time or using a different network connection.
```
